tests/jest/aggregate-themes.test.js

```diff
@@ -4,6 +4,37 @@ const ThemeAggregator = require('../../scripts/aggregate-themes/theme-aggregator
 global.console = { log: jest.fn(), warn: jest.fn() };
 
 describe('Theme Aggregator', () => {
+  /* eslint-disable no-console */
+  describe('validate', () => {
+    it('does not warn if there is a default theme but not a scoped theme', () => {
+      const options = { theme: 'terra-mock-dark-theme' };
+
+      ThemeAggregator.validate(options);
+
+      expect(console.warn).toHaveBeenCalledTimes(0);
+    });
+
+    it('does not warn if there is a scoped theme but not a default theme', () => {
+      const options = { scoped: ['terra-mock-dark-theme'] };
+
+      ThemeAggregator.validate(options);
+
+      expect(console.warn).toHaveBeenCalledTimes(0);
+    });
+
+    it('warns if there is not a default theme or a scoped theme', () => {
+      const options = {};
+
+      ThemeAggregator.validate(options);
+
+      expect(console.warn).toHaveBeenCalled();
+    });
+
+    console.warn.mockClear();
```

Comment:
This need to be wrapped in an `afterAll` block to be hit

Reply:
If we wrap in an `afterAll`, we reset the mocks after all tests are complete - perhaps `afterEach` instead?